### DISCUSSION TITLE
DataGrid EmptyTemplate doesnt work

### DISCUSSION THREAD

**Discussion by yangetze** [Megabit/Blazorise#4410] [Q&A] [answered]
Today I updated to v1.1.5 and now EmptyTemplate doesn't show anything.
I have my own component to show inside this template, but even removing it, does not show any message in this area.  
This is datagrid component. 
What I missing?

```
<EmptyTemplate>
             <DataGridEmpty Module="@_componentTitle" CreateNewItem="CreateBank" />
 </EmptyTemplate>
```

**Comment by stsrki** [comment, marked as answer]
Sorry for that. The good news is that we have already fixed it in #4390. It will be released soon.
